packages/web3/src/api/api-alephium.ts

```diff
@@ -300,12 +288,35 @@ export interface BuildExecuteScriptTx {
   gasPrice?: string
   /** @format block-hash */
   targetBlockHash?: string
+  /** @format group-index */
+  group?: number
   /** @format double */
   gasEstimationMultiplier?: number
 }
 
 /** BuildExecuteScriptTxResult */
-export interface BuildExecuteScriptTxResult {
+export type BuildExecuteScriptTxResult = BuildGrouplessExecuteScriptTxResult | BuildSimpleExecuteScriptTxResult
+
+/** BuildGrouplessDeployContractTxResult */
+export interface BuildGrouplessDeployContractTxResult {
+  /** @format int32 */
+  fromGroup: number
+  /** @format int32 */
+  toGroup: number
+  unsignedTx: string
+  /** @format gas */
+  gasAmount: number
+  /** @format uint256 */
+  gasPrice: string
+  /** @format 32-byte-hash */
+  txId: string
+  /** @format address */
+  contractAddress: string
+  transferTxs: BuildSimpleTransferTxResult[]
```

Comment:
As discussed:
1. We'll rename this to `fundingTxs`
2. We'll make this field optional too